frontend/src/pages/patients/PatientsPage.tsx

```diff
@@ -15,7 +15,7 @@ import InfoIcon from "@material-ui/icons/InfoOutlined";
 import { parseUserSearchVal } from "../../utils/parseSearchQueries";
 import {
   PatientsListColumns,
-  SampleDetailsColumns,
+  SampleMetadataDetailsColumns,
```

Comment:
Just a rename. We have a `CohortSampleDetailsColumns` so I wanted to specify that this variable is for SampleMetadata.